src/ceph-daemon/ceph-daemon

```diff
@@ -1915,6 +1992,14 @@ def _get_parser():
         '--osd-fsid',
         help='OSD uuid, if creating an OSD container')
 
+    parser_check_host = subparsers.add_parser(
+        'check-host', help='check host configuration')
```

Comment:
@swiftgist can we make use of this?

Reply:
As a blanket check called against all the minions?  sure... the out-of-scope question on this PR, how do we handle failed checks?  That's more of a prerequisite topic of whether to address all/any steps ahead of time or let the admin resolve them.

Reply:
Do we need to differ between warnings and errors? Would that help?